tools/os_tools.go

```diff
@@ -29,6 +29,13 @@ func Getwd() (dir string, err error) {
 
 func translateCygwinPath(path string) (string, error) {
 	cmd := subprocess.ExecCommand("cygpath", "-w", path)
+	// cygpath uses ISO-8850-1 as the default encoding if the locale is not
+	// set, resulting in breakage, since we want a UTF-8 path.
+	env := make([]string, 0, len(cmd.Env)+1)
+	for _, val := range cmd.Env {
+		env = append(env, val)
+	}
+	cmd.Env = append(env, "LC_ALL=C.UTF-8")
```

Comment:
I would imagine that by appending this at the end of the `env` array, if `LC_ALL` appears earlier as well, it will be supplanted by this value ... I'm virtually certain that would be the effect on Unix, but a little less so about Windows.  Might it be useful to adjust the test to explicitly try setting `LC_ALL` to something else before running `git lfs env`?

Reply:
Good point.  I'll just strip out the `LC_ALL` entry if it exists.